File: app/src/main/java/igrek/todotree/model/treeitem/SeparatorTreeItem.java
package igrek.todotree.model.treeitem;


public class SeparatorTreeItem extends AbstractTreeItem {
	
	public SeparatorTreeItem(AbstractTreeItem parent) {
		super(parent);
	}
	
	@Override
	public SeparatorTreeItem clone() {
		return new SeparatorTreeItem(null).copyChildren(this);
	}
	
	@Override
	public String getDisplayName() {
		return "----------";
	}
	
	@Override
	public String getTypeName() {
		return "separator";
	}
}

File: app/src/main/java/igrek/todotree/domain/treeitem/SeparatorTreeItem.kt
package igrek.todotree.domain.treeitem

class SeparatorTreeItem(parent: AbstractTreeItem?) : AbstractTreeItem(parent) {
    override fun clone(): SeparatorTreeItem {
        return SeparatorTreeItem(null).copyChildren(this)
    }

    override val displayName: String
        get() = "----------"

    override val typeName: String
        get() = "separator"
}